### PR TITLE
Loader page ordering

### PR DESCRIPTION
**Pull request by kuiche**
Allows us to set the order in which pages may return

### CODE REVIEW

**Comment by thomasjthomasj** [comment]
The ID ordering breaks because `page_id` is ambiguous

**Comment by thomasjthomasj** [comment]
I'm not sure about the 'natural' ones, as it's not clear what 'natural' means, could we not just call them `DEFAULT` and `REVERSE`?

Also I'm generally of the opinion that the names could be a bit more natural and less 'techy'. The word 'order' is tricky because while it IS affecting the order, an 'order' also exists as a model elsewhere in the system, and when I see a constant at a glance that starts with `ORDER`, I instantly assume it relates back to the commerce module and that's it an event or something.

Could we perhaps make a class to hold these constants? Also maybe call the `setOrder()` method `orderBy()` instead? So the syntax could be something like `->orderBy(LoadOrder::CREATED_AT_ASC)`? I'd personally find that quite a bit clearer and easier to read

**Comment by kuiche** [comment]
I agree, I think I'll make a new class for holding them.

File: src/Page/PageOrder.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ */
+class PageOrder
+{
+	const CREATED_DATE         = "order.date.created.asc";
+	const CREATED_DATE_REVERSE = "order.date.created.desc";
+	const UPDATED_DATE         = "order.date.updated.asc";
+	const UPDATED_DATE_REVERSE = "order.date.updated.desc";
+	const ID                   = "order.id.asc";
+	const ID_REVERSE           = "order.id.desc";
+	const DEFAULT              = "order.natural.asc";
```
Comment (thomasjthomasj) [inline src/Page/PageOrder.php]: DEFAULT is a reserved constant, use STANDARD instead

**Pull request merged**